state:

  # Upper and lower bounds for experimental space (meters). [x, y, z]
  lower: [-325, -250, 0] 
  upper: [325, 250, 2.0] 

pred:
  # Total number of humans that you will be simulating. Used 
  # by potential_field_human.py to avoid other humans.
  total_number_of_humans: 3

  # Start position of human(s) and list of potential goal locations
  human1_real_start: [-14, 11] 
  human1_real_goals: [[-13, -10]] 

  human2_real_start: [-13, -10] 
  human2_real_goals: [[-13, 10]] 

  human3_real_start: [0, 11] 
  human3_real_goals: [[-14, -0]] 

  # color coding for human visualization (RGB)
  human1_color: [0.0, 0.0, 1.0]
  human2_color: [0.0, 0.0, 1.0]
  human3_color: [1.0, 0.0, 0.0]

  # Dimensions of the prediction map
  sim_height: 1300
  sim_width: 1000

  # Resolution, ex. (real meters)/(sim dim-1) (m/cell)
  resolution_x: 0.5 #0.1464 
  resolution_y: 0.5

  # Num timesteps into the future to predict
  fwd_tsteps: 10

  # Initial betas depending on model
  beta_adaptive: [0.1, 0.3, 1.0, 3.0, 10.0]
  beta_rational: [0.05]
  beta_irrational: [10.0]

  # Human's height (in meters)
  human_height: 1.67

  # Human's walking speed (m/s) -- human walking speed is 1.4 m/s
  human_vel: 0.7 

  # Probability threshold 
  prob_thresh: 0.01

  # Process noise (epsilon-stubborn model)
  epsilon_dest: 0.02
  epsilon_beta: 0.02
